### ISSUE TITLE
Feature: Remember GUI Window Size

### ISSUE THREAD

**Issue by PromontoryProtean** [webprofusion/certify#417]
This is a minor quality-of-life request, but it would be useful if the Certify GUI window would remember its size after it has been resized and closed. I find myself resizing the window every time I use it, and when maintaining hundreds of sites it would be nice if this would be remembered.
Thanks,
John

**Comment by webprofusion-chrisc** [comment]
Thanks, this is a dupe of #318.

**Comment by PromontoryProtean** [comment]
Sorry, I did a search before I posted but did not find that entry. I would like to take a shot at adding this in a PR, but I won't be able to until the end of the month because I am in the process of moving my entire company into Azure. So if it is still up for grabs later this month I will clone the repo and take a look through.

Do you want to add this as a toggle-able setting? Or should it be done without user intervention?

**Comment by webprofusion-chrisc** [comment]
@PromontoryProtean that would be great, I don't think it would require a setting in the UI but as we have no other dependencies on the registry it would likely be stored as json on app close and read again on main window loaded. I'd be happy if we started an independent UI settings file, as updating the current preferences/settings stuff is quite complicated for legacy reasons (we used to use .net app settings and migrated from that).

`Certify.Management.Util.GetAppDataFolder();` will give you the c:\programdata\certify\ path.

**Comment by PromontoryProtean** [comment]
@webprofusion-chrisc I finally had time to take a look at the repo. I just created a pull request. Hopefully it lines up with what you had in mind. Just let me know if you would like to see it implemented differently and I will change it up. I went with a very simple approach, but we could create a more complex ui settings service or something like that if you anticipate adding future capabilities for UI preferences.
